### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: '/home/user/.seleniumwire/storage ...

### ISSUE THREAD

**Issue by tsoernes** [wkeeling/selenium-wire#284]
```
File "/home/torstein/anaconda3/lib/python3.8/site-packages/IPython/core/interactiveshell.py", line 3437, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
File "<ipython-input-3-ecd9de9f6ad5>", line 1, in <module>
    import codecs, os;__pyfile = codecs.open('''/tmp/pyd7lAB6''', encoding='''utf-8''');__code = __pyfile.read().encode('''utf-8''');__pyfile.close();os.remove('''/tmp/pyd7lAB6''');exec(compile(__code, '''/home/torstein/code/fintechdb/Jobs/glassdoor/get_jobtitles_gql.py''', 'exec'));
File "/home/torstein/code/fintechdb/Jobs/glassdoor/get_jobtitles_gql.py", line 35, in <module>
    driver, result = get_result(driver, id_)
File "/home/torstein/code/fintechdb/Jobs/glassdoor/get_jobtitles_gql.py", line 26, in get_result
    return make_query(driver, token, body(id_))
File "/home/torstein/code/fintechdb/Jobs/glassdoor/utils.py", line 37, in make_query
    driver, token = get_driver()
File "/home/torstein/code/fintechdb/Jobs/glassdoor/get_driver.py", line 21, in get_driver
    lambda r: "gd-csrf-token" in r.headers, reversed(driver.requests)
File "/home/torstein/anaconda3/lib/python3.8/site-packages/seleniumwire/webdriver/request.py", line 25, in requests
    return self.proxy.storage.load_requests()
File "/home/torstein/anaconda3/lib/python3.8/site-packages/seleniumwire/proxy/storage.py", line 126, in load_requests
    request = self._load_request(indexed_request.id)
File "/home/torstein/anaconda3/lib/python3.8/site-packages/seleniumwire/proxy/storage.py", line 134, in _load_request
    with open(os.path.join(request_dir, 'request'), 'rb') as req:

FileNotFoundError: [Errno 2] No such file or directory: '/home/torstein/.seleniumwire/storage-c8f37b76-374d-4083-b0ef-c21f6b849640/request-60b2b665-f487-453f-9a86-92352b3a4ca6/request'
```
seleniumwire.__version__  '3.0.5'

**Comment by wkeeling** [comment]
This error is due to a concurrency issue that was fixed in 4.2.2. Are you able to upgrade?

**Comment by tsoernes** [comment]
Yes.
